aptos-move/block-executor/src/scheduler.rs

```diff
@@ -343,7 +342,6 @@ impl Scheduler {
 
     /// Return the next task for the thread.
     pub fn next_task(&self, committing: bool) -> SchedulerTask {
-        let _timer = GET_NEXT_TASK_SECONDS.start_timer();
```

Comment:
Good idea to remove this from the hottest function.  We can possibly add a conditionally compiled perf timer, or something similar.

Reply:
This counter is not very useful anymore anyways.. so let's just get rid of this.